static/js/message_list_data.js

```diff
@@ -166,7 +166,7 @@ MessageListData.prototype = {
 
     unmuted_messages: function (messages) {
         return _.reject(messages, function (message) {
-            return muting.is_topic_muted(message.stream, message.subject) &&
+            return muting.is_topic_muted(message.stream_id, message.subject) &&
```

Comment:
Do message objects have a `.stream_id` property always when they have a `.stream`?

Reply:
Yes, we've been putting `stream_id` on message objects for a while:

https://github.com/zulip/zulip/blob/master/zerver/lib/message.py#L460